django/db/backends/base/base.py

```diff
@@ -44,6 +44,12 @@ class BaseDatabaseWrapper(object):
 
     queries_limit = 9000
 
+    set_operators = {
```

Comment:
Did you consider putting this on the `DatabaseOperations` class? Besides the Oracle operators stuff, it seems like it might be a better fit there. (I could make the change as I'm revising the patch now.)